### PR TITLE
Rebuild /internet-of-things/robotics page

### LINKED ISSUE #5638: Rebuild of /internet-of-things/robotics
Please update the robotics page based on this [copy doc](https://docs.google.com/document/d/1cDxhPCQbHA0r6n9JXNmO50ANLPZhCFdgwATX6f90guU/edit#)

Note:

- it is missing many illustrations, these will come later

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Add `/internet-of-things/robotics` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/internet-of-things/robotics](http://0.0.0.0:8001/internet-of-things/robotics)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [copy doc](https://docs.google.com/document/d/1cDxhPCQbHA0r6n9JXNmO50ANLPZhCFdgwATX6f90guU/edit#)

## Issue / Card

Fixes #5638

### CODE REVIEW

File: templates/internet-of-things/robotics.html [hunk 1]
```diff
@@ -4,148 +4,314 @@
...
 </div>
 
-<div class="p-strip--accent is-bordered">
+<section class="p-strip--accent is-bordered">
   <div class="row">
-    <div class="col-10 prefix-1">
+    <div class="col-12">
+      <p class="p-heading--two">Enhanced security for mission critical robots</p>
+      <p>Several layers of security are stacked in Ubuntu Core to keep potential surfaces of attack as low as reasonably possible for autonomous robots deployed in field of operations.</p>
+      <a class="p-link--inverted" href="{{ ASSET_SERVER_URL }}66fcd858-ubuntu-core-security-whitepaper.pdf">Whitepaper - Ubuntu Core security&nbsp;›</a>
```
Comment (pmahnke) [inline templates/internet-of-things/robotics.html]:
```suggestion
      <a class="p-link--inverted" href="{{ ASSET_SERVER_URL }}66fcd858-ubuntu-core-security-whitepaper.pdf">Read the Ubuntu Core security whitepaper&nbsp;&rsaquo;</a>
```

File: templates/internet-of-things/robotics.html [hunk 2]
```diff
@@ -4,148 +4,314 @@
...
+    </div>
+  </div>
+</section>
+
+<section class='p-strip'>
+  <div class="row">
+    <div class="col-6">
+      <h4>Largest pool of talent</h4>
+      <p>Developers love Ubuntu. By adopting the most popular Linux distribution, you can leverage the strong community of developers and roboticists around Ubuntu.</p>
+      <a href="/engage/small-robot-company">Read the case study - Small Robot Company&nbsp;›</a>
```
Comment (pmahnke) [inline templates/internet-of-things/robotics.html]:
```suggestion
      <a href="/engage/small-robot-company">Read the Small Robot Company case study&nbsp;&rsaquo;</a>
```

File: templates/internet-of-things/robotics.html [hunk 3]
```diff
@@ -4,148 +4,314 @@
...
-<div class="p-strip is-deep is-bordered">
+  <div class="row">
+    <hr class="p-separator" />
+  </div>
+
+  <div class="row">
+    <div class="col-6">
+      <h4>Open source software</h4>
+      <p>Complex software is increasingly collaborative and open source. This was our philosophy from the beginning.  Harness the power of our open source software and build on top of contributions from the community. Build on your own, or seek our help. We will support you.</p>
+      <a href="/engage/northstar">Read the case study - Northstar Robotics&nbsp;›</a>
```
Comment (pmahnke) [inline templates/internet-of-things/robotics.html]:
```suggestion
      <a href="/engage/northstar">Read the Northstar Robotics case study&nbsp;&rsaquo;</a>
```

File: templates/internet-of-things/robotics.html [hunk 4]
```diff
@@ -4,148 +4,314 @@
...
+</section>
+
+<section class="p-strip--accent is-bordered">
   <div class="row">
     <div class="col-12">
-      <h2>Learn more about robotics</h2>
+      <p class="p-heading--two">Catalyses new business models </p>
+      <p>Improve the return on investment for your robotic assets.</p>
+      <p class="p-link--external">
+        <a class="p-link--inverted" href="https://www.brighttalk.com/webcast/6793/290087">Watch the app store for Linux webinar</a>
```
Comment (pmahnke) [inline templates/internet-of-things/robotics.html]:
```suggestion
        <a class="p-link--inverted p-link--external" href="https://www.brighttalk.com/webcast/6793/290087">Watch the app store for Linux webinar</a>
```

File: templates/internet-of-things/robotics.html [hunk 5]
```diff
@@ -4,148 +4,314 @@
...
-      <h3 class='p-heading--four'><a class='p-link--external' href='/blog/your-first-robot-a-beginners-guide-to-ros-and-ubuntu-core-1-5' onclick='dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'product page', 'eventAction' : 'gateways - right', 'eventLabel' : 'Article - Build your first robot with ROS and Ubuntu Core', 'eventValue' : '1' });'><span hidden>Read the article </span>Build your first robot with ROS and Ubuntu Core</a></h3>
-      <!-- rtp section end -->
+</section>
+
+<section class='p-strip'>
+  <div class="row">
+    <div class="col-6">
+      <h4>New avenues for monetisation</h4>
+      <p>Public and private app stores unlock new avenues for monetisation. Earn recurring revenue, upsell add-ons to your users, or create platform ecosystems around your robots.</p>
+      <a href="/internet-of-things/appstore">Learn more about the Snap store</a>
```
Comment (pmahnke) [inline templates/internet-of-things/robotics.html]:
```suggestion
      <a href="/internet-of-things/appstore">Learn more about the Snap store&nbsp;&rsaquo;</a>
```

File: templates/internet-of-things/robotics.html [hunk 6]
```diff
@@ -4,148 +4,314 @@
...
+      <p class="p-card__desc">Deep learning machine vision applications can be developed with OpenCV on top of Ubuntu.</p>
+    </div>
+    <div class="p-card col-4">
+      <h3 class="p-card__title">PX4</h3>
+      <p class="p-card__desc">Ubuntu Core is the best fit for autonomous systems. Snaps allow to orchestrate the different components of the PX4 library to develop aerial robotic systems</p>
+    </div>
+  </div>
+  <div class="row u-sv3">
+    <div class="col-12">
+      <p><a class="p-button--positive" href="https://snapcraft.io/blog/your-first-robot-a-beginners-guide-to-ros-and-ubuntu-core-1-5">Build your robot with ROS and Ubuntu Core</a></p>
```
Comment (pmahnke) [inline templates/internet-of-things/robotics.html]:
```suggestion
      <p><a class="p-button--positive" href="https://snapcraft.io/blog/your-first-robot-a-beginners-guide-to-ros-and-ubuntu-core-1-5"><span class="p-link--external">Build your robot with ROS and Ubuntu Core</span></a></p>
```

File: templates/internet-of-things/robotics.html [hunk 7]
```diff
@@ -4,148 +4,314 @@
...
+    </div>
+    <div class="p-card col-4">
+      <header class="p-card__header">
+        <h4>Dragonboard</h4>
+      </header>
+      <div class="p-card__content">
+        <img src="{{ ASSET_SERVER_URL }}a64b7ee2-dragonboard.png" alt="" style="padding-top: 1rem;">
+      </div>
     </div>
   </div>
```
Comment (pmahnke) [inline templates/internet-of-things/robotics.html]:
```suggestion
  <p><a href="https://ubuntu.com/blog/porting-ubuntu-core-18-to-nvidia-jetson-tx1-developer-kit">Read more about building Ubuntu Core for NVIDIA Jetson&nbsp;&rsaquo;</a></p>
  </div>
```

File: templates/internet-of-things/robotics.html [hunk 8]
```diff
@@ -4,148 +4,314 @@
...
+    <div class="u-equal-height">
+      <div class="col-4 u-align--center u-hide--small u-vertically-center">
+        <img src="https://assets.ubuntu.com/v1/1f1d581a-picto-quote-orange.svg" width="200" alt="">
+      </div>
+      <div class="col-7 prefix-1 u-vertically-center">
+        <div class="row">
+          <div class="col-12">
+            <h2>Ready for the next steps?</h2>
+            <p>Roll out your robots with Ubuntu Core.</p>
+            <!-- #################### Need a link ink below ############# -->
```
Comment (pmahnke) [inline templates/internet-of-things/robotics.html]:
```suggestion
```

File: templates/internet-of-things/robotics.html [hunk 9]
```diff
@@ -4,148 +4,314 @@
...
+            <p><a class="p-button--positive js-invoke-modal" href="/kubeflow/contact-us?product=ai-install">Get in touch</a></p>
```
Comment (pmahnke) [inline templates/internet-of-things/robotics.html]:
```suggestion
            <p><a class="p-button--positive js-invoke-modal" href="/internet-of-things/contact-us?product=robotics"">Get in touch</a></p>
```

File: templates/internet-of-things/robotics.html [hunk 10]
```diff
@@ -4,148 +4,314 @@
...
+
+<section class='p-strip'>
+  <div class="row">
+    <div class="col-6">
+      <h4>New avenues for monetisation</h4>
+      <p>Public and private app stores unlock new avenues for monetisation. Earn recurring revenue, upsell add-ons to your users, or create platform ecosystems around your robots.</p>
+      <a href="/internet-of-things/appstore">Learn more about the Snap store</a>
+    </div>
+    <div class="col-6 u-hide--small u-align--center">
+      <img src="{{ ASSET_SERVER_URL }}fa103ed7-1A-shield.svg" width="185" alt="">
```
Comment (pmahnke) [inline templates/internet-of-things/robotics.html]:
```suggestion
      <img src="{{ ASSET_SERVER_URL }}682f351e-Faster_cheaper.svg " width="185" alt="">
```

File: templates/internet-of-things/robotics.html [hunk 11]
```diff
@@ -4,148 +4,314 @@
...
+  </div>
+
+  <div class="row">
+    <div class="col-6">
+      <h4>Enabling robots as a platform</h4>
+      <p>Our brand stores will allow you to commercialise your robots in a way that was not possible before. You will tie in multiple suppliers and distributors to your product ecosystem. The future proof technology underlying snaps will stretch the useful life of your robots, and transform them into evolutive platforms.</p>
+      <a href="/engage/iot-business-model">Learn more about brand stores</a>
+    </div>
+    <div class="col-6 u-hide--small u-align--center">
+      <img src="{{ ASSET_SERVER_URL }}fa103ed7-1A-shield.svg" width="185" alt="">
```
Comment (pmahnke) [inline templates/internet-of-things/robotics.html]:
```suggestion
      <img src="{{ ASSET_SERVER_URL }}8ccd2dd7-3A-One+platform.svg" width="185" alt="">
```

File: templates/internet-of-things/robotics.html [hunk 12]
```diff
@@ -4,148 +4,314 @@
...
-<div class="p-strip--light is-deep is-bordered">
+  <div class="row">
+    <hr class="p-separator" />
+  </div>
+
+  <div class="row">
+    <div class="col-6">
+      <h4>Enabling robots as a platform</h4>
+      <p>Our brand stores will allow you to commercialise your robots in a way that was not possible before. You will tie in multiple suppliers and distributors to your product ecosystem. The future proof technology underlying snaps will stretch the useful life of your robots, and transform them into evolutive platforms.</p>
+      <a href="/engage/iot-business-model">Learn more about brand stores</a>
```
Comment (pmahnke) [inline templates/internet-of-things/robotics.html]:
```suggestion
      <a href="/engage/iot-business-model">Learn more about brand stores&nbsp;&rsaquo;</a>
```

**Comment by pmahnke** [comment]
@anasereijo I think the robot should float up higher on the banner  what do you think?

image

**Comment by anasereijo** [comment]
@solazio 

- [ ] as @pmahnke said the robot should be higher up on the banner. Move the title to the same col as the text and vertically centre the image

- [ ] Make "Enhanced security for mission critical robots" section light grey

- [ ] "System integrity, assured" and all other titles at this level are `h4` and should be `h3` as per copydoc

- [ ] Make "The strongest robot developers community" section light grey

- [ ] Make "Catalyses new business models" light grey

- [ ] Make "Ready for all boards, frameworks and applications" section light grey

- [ ] Make "Bootstrap for production-grade robots" section light grey

- [ ]

**Pull request merged**